autogen/main/variables.tf.tmpl

```diff
@@ -243,6 +243,8 @@ variable "cluster_autoscaling" {
     gpu_resources       = list(object({ resource_type = string, minimum = number, maximum = number }))
     auto_repair         = bool
     auto_upgrade        = bool
+    disk_size           = optional(number)
+    disk_type           = optional(string)
```

Comment:
Since we are using optional, could you also update min TF version to `>= 1.3`
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/f077184f735c180d98f28046c0ce25fa331805af/autogen/main/versions.tf.tmpl#L21
https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/f077184f735c180d98f28046c0ce25fa331805af/autogen/safer-cluster/versions.tf.tmpl#L23

Reply:
Thanks, done.